### ISSUE TITLE
Migrate to leaflet 1.x

### ISSUE THREAD

**Issue by rochoa** [CartoDB/Leaflet.CanvasLayer#23]

**Comment by pabletecodes** [comment]
We should consider this change too:
https://github.com/CartoDB/torque/pull/285/files#diff-1b906f3556a080f21262ae9dfe1b4639

**Comment by pabletecodes** [comment]
We still haven't merged https://github.com/CartoDB/Leaflet.CanvasLayer/pull/24 @donflopez. We should include what I mentioned in my previous comment. Also, should we use this in Torque as @rochoa suggested at some point?

**Comment by donflopez** [comment]
Oops, I thought it was cartodb.js

**Comment by donflopez** [comment]
yes, but maybe we should create another issue and prioritise it. I also saw code diffs in torque that aren't here.

**Comment by rochoa** [comment]
@donflopez do you think this is important now or should we wait for using it as a real dependency in the future?

**Comment by donflopez** [comment]
I think that we can wait to make a check over all of those libraries being used with c&p in cartodb.js @rochoa
